static/base.css

```diff
@@ -1,3 +1,6 @@
+
+a {text-decoration: none;}
```

Comment:
This is here to prevent all our tables from having underlines under things like names of persons.